setup.py

```diff
@@ -24,6 +23,9 @@
 if os.path.exists("./prisma"):
     include_files.append(("./prisma", "prisma"))
 
+if os.path.exists("./database.db"):
```

Comment:
Also, try to include schema.prisma in lib/prisma/schema.prisma

In the site-packges/prima/client.py there are some hardcoded paths generated by db push.